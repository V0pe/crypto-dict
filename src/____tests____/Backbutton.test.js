import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import BackButton from '../Components/Backbutton';
import store from '../Redux/Configurestore';

it('renders the snpashot correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><BackButton /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
