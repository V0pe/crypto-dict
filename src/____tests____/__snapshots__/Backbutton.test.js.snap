// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the snpashot correctly 1`] = `
<div>
  <button
    className="logoButton"
    onClick={[Function]}
    type="button"
  >
    <h1>
      <img
        alt="logo"
        className="app-logo"
        src="cryptocurrencies.png"
      />
    </h1>
  </button>
</div>
`;
